src/python/pybind11/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if (PythonLibs_FOUND)
     SET(PKGDIR "site-packages")
   endif(${LINUX_FLAVOR} MATCHES "^(Ubuntu|Debian)")
 
-  find_package(pybind11 2.6.0 PATHS "/usr/local/lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/${PKGDIR}/pybind11")
+  find_package(pybind11 2.6.0 REQUIRED PATHS "/usr/local/lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/${PKGDIR}/pybind11")
```

Comment:
Hmm...Are PATHS still required?

Reply:
**REQUIRED** essentially will fail XRT build if the minimum requested version is not found. pybiind11 is required (not optional) since our 22_verify.py test distributed with `xbutil validate` relies of pybind11 binding of XRT. Note the pybind11 package requirement is XRT compile time requirement and not a runtime requirement.

Reply:
What's the deal with 'local' as in '/usr/local'?    Is that always where pybind11 will be installed?
Is pip3 the proper way to install pybind11?  And will that always install under /usr/local ?
I think 18.04 and later support apt install python3-pybind11 ?

Reply:
pip3 is installing pybind11 to /usr/local. On all platforms we are getting >= version 2.6.0
apt install is providing an older version pybind11 (< version 2.6.0) which is not usable because of bugs related to stale format string pointer in memoryview from buffer constructor.